tests/onegov/election_day/common.py

```diff
@@ -17,6 +19,57 @@ def print_errors(errors):
         print(f'{fn}:{l} {err}')
 
 
+def get_fixture_path(domain=None, principal=None):
+    """Fixtures are organized like
+    fixtures/domain/principal/internal_proporz.tar.gz
+    """
+    fixture_path = module_path('tests.onegov.election_day', 'fixtures')
+    if not domain:
+        return fixture_path
+    if not principal:
+        return os.path.join(fixture_path, domain)
+    return os.path.join(fixture_path, domain, principal)
+
+
+def get_tar_archive_name(api_format, model, election_type=None):
+    if model == 'vote':
+        return f'{api_format}_vote.tar.gz'
+    elif model == 'election':
+        assert election_type
+        return f'{api_format}_{election_type}.tar.gz'
+
+
+def get_tar_file_path(
+        domain=None, principal=None,
+        api_format=None,
+        model=None,
+        election_type=None
+):
+    if model == 'vote' and api_format == 'wabstic' or api_format == 'wabstim':
+        # This format can have all domains, the will be a separate archive
+        return os.path.join(
+            get_fixture_path(),
+            f'{api_format}_vote.tar.gz'
+        )
+    return os.path.join(
+        get_fixture_path(domain, principal),
+        get_tar_archive_name(api_format, model, election_type)
+    )
+
+
+def create_principal(principal=None, municipality=None):
+    if principal in Canton.CANTONS:
+        pr = Canton(canton=principal)
+    else:
+        pr = Municipality(municipality=municipality)
+    return pr
+
+
+def compare_elections(left, right):
```

Comment:
Since this doesn't actually do anything, you should either remove it or raise a `NotImplementedError`

Reply:
removed...